snyk.go

```diff
@@ -30,23 +31,23 @@ func getOrgProjects(flags flags, customDebug debug) (jsn.Json, error) {
 	// a POST API call but historically we used GET here. The following code maintains backwards compatibility for
 	// existing cases where people aren't filtering projects by attributes, as it appears the API does not return
 	// the full project list with empty attribute filters in the request body.
-	verb := "GET"
+	verb := "POST"
```

Comment:
Why changing this?

Reply:
it will always be a POST now because we always have to send isMonitored in the body